xcube_cci/dataaccess.py

```diff
@@ -424,7 +426,7 @@ def get_search_params_schema(cls, data_type: str = None) -> JsonObjectSchema:
                           .replace('-yrs', ' years').replace('yr', 'year')
                            for data_id in data_ids])
         processing_levels = set([data_id.split('.')[3] for data_id in data_ids])
-        data_types = set([data_id.split('.')[4] for data_id in data_ids])
+        types_of_data = set([data_id.split('.')[4] for data_id in data_ids])
```

Comment:
A much cleaner approach would be to have a search parameter `attrs` that may include search attributes and where the attribute  `data_type` is included as specified by the ODP. 

The names `data_type` and `types_of_data ` side-by-side as search parameters is hard to explain to users (and to me).

Reply:
I agree with forman :) would prefer a search param including attrs